my-dsl.md

```diff
@@ -1,38 +1,73 @@
 # Language
 _What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+-(if appropriate)._
+
+Django--https://www.djangoproject.com/
 
 
 # Domain
 _Describe the language's domain in five words._
 
+Web development for Python users.
+
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
-For now, try to come up with the clearest, most concise explanation of 
-what happens when a program in your DSL runs._
+-For now, try to come up with the clearest, most concise explanation of 
+-what happens when a program in your DSL runs._
+
+Django is compiled through the command line, like other Python programs.After 
+that, the code is by default sent to the Django development server, where they 
+can view it. Users could also redirect the code to their website when they are 
+finished developing.
 
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
-"purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
-and why?_ 
+-"purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
+-and why?_ 
+
+Django sits right in the middle of the spectrum. Fowler mentions limited
+expressiveness, espeically for internal DSLs. On page 30, he notes that "it's
+common to avoid conditions, looping constructs, and variables." Django includes 
+conditional statements and relies heavily on variables. Additionally,
+everything is broken down into classes and methods, which are rather abstract.
+However, Django includes many helpful classes, like models, which allow for the
+feel of "putting together full sentences," like Fowler mentions on page 30. For
+instance, the class "models" gives methods like DateTimeField, which build in
+necessary elements for web development.
```

Comment:
I'm still of the same mind as before - this isn't that convincing. I think Django as a framework is great, but it isn't a DSL. I think you could (very easily) make the argument that templates (i.e. Jinja2 templates) are a DSL, but I'm not convinced by Django as a whole.